CONTRIBUTING.md

```diff
@@ -240,7 +240,7 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
 
 * Julia
   - **Homepage:** <http://julialang.org>
-  - **Mailing lists:** <http://julialang.org/community/>
+  - **Discussion:** <http://julialang.org/community/>
```

Comment:
Maybe simply call it *Community*, since neither `Mailing lists` nor `Discussion` is quite right